src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public enum Color {
```

Comment:
The enum 'Color' should be declared in its own file or at least outside the Ball class if it's meant to be used by other classes as well. This will make the code more organized and the enum easier to reuse.